packages/core/src/messaging/eventBus.ts

```diff
@@ -1,8 +1,6 @@
 import { EventEmitter } from "eventemitter3";
 import type { Logger } from "../logging/logger.ts";
 
-export type EventName = string | symbol;
```

Comment:
I there a specific reason to not allow `symbol` anymore? It might be what we want to go for as it ensure uniqueness.